config/goGenerate.coffee

```diff
@@ -0,0 +1,18 @@
+module.exports = (options, credentials) ->
```

Comment:
@szkl you are right. I followed naming conversions I see files like `workers.cofffee` which also returns an object. According to this, what about `goConfig.coffee` or `generateGoConfig.coffee` (however the former includes a `generate` verb which suggests that we do some actions inside). I'd rather not use `generage-go.coffee` since it's inconsistent with other file names in `config` directory.